tensorflow/core/tpu/tpu_initializer_helper.cc

```diff
@@ -113,15 +113,14 @@ bool FindAndLogLibtpuProcess() {
 
     pid = strtol(ent->d_name, nullptr, 10);
     if (IsTpuUsed(pid)) {
-      LOG(INFO) << "libtpu.so is already in use by process with pid " << pid
-                << ". Not attempting to load libtpu.so in this process.";
-      return true;
-    }
+      std::string error_message = "libtpu.so is already in use by process with pid " + std::to_string(pid) + ". Not attempting to load libtpu.so in this process.";
+      return error_message;
+   }
   }
-  return false;
+  return "";
 }
 
-bool TryAcquireTpuLock() {
+Status TryAcquireTpuLock() {
   static absl::Mutex* mu = new absl::Mutex();
   absl::MutexLock l(mu);
 
```

Comment:
Get rid of attempted_file_open and should_load_library, neither of them actually do anything!